### ISSUE TITLE
Accessibility: SplitButton is not functional for screen reader users and sighted keyboard-only users

### ISSUE THREAD

**Issue by mitchellevan** [telerik/kendo-angular#3265]
**Describe the bug**

The SplitButton has invalid and misleading WAI-ARIA and incomplete keyboard behaviors.

Issues with the textual first button:

* ARIA `role=listbox` fails HTML validation on the `<button>` element and prevents screen reader users from understand the function of this control.
* The `aria-owns` attribute is inappropriate. It currently points to a nonexistent id. The `aria-owns` attribute is unsupported in most screen readers. The `aria-owns` attribute in theory creates a child relationship, which would be inappropriate for a singleton button. 
* The button's accessible name includes the word "splitbutton". While this conveys to advanced users of Windows screen readers that it will respond to Alt+DownArrow, this interaction is not familiar to most screen reader users with Windows browsers, and is not an accessible solution for non-Windows browsers.
* It is not evident to sighted keyboard users how to operate this control by keyboard. [Developer documentation for the custom Alt+DownArrow keystroke](https://www.telerik.com/kendo-angular-ui/components/buttons/splitbutton/keyboard-navigation/) is not provided to end users in context. Advanced keyboard users on Windows might guess that this is a split button, even though the visual focus indicator wraps around only the first of the two buttons (unlike a standard Windows OS native split button, where the visual focus indicator wraps around both halves of the split button). Non-Windows users are not likely to be familiar with this interaction pattern.

Issues with the graphical second button:

https://github.com/telerik/kendo-angular/issues/2139#issuecomment-498226467 accurately states that tabbing skips the second button; however, it is not true that this hides the button from screen readers. When a screen reader user reaches this control by arrow navigation, the button has no accessible name and does not convey its expandable function and open/closed state.

Issues with the pop-up:

* The pop-up is not in logical consecutive DOM reading order relative to the button which triggers it.
* The pop-up has an incorrectly implemented ARIA menu pattern.
* Enter and Space do not activate items in the list. Reproducible both with a screen reader and without a screen reader (sighted keyboard-only use).

**To Reproduce**

https://www.telerik.com/kendo-angular-ui/components/buttons/splitbutton/

Use a compatible screen reader with the Chrome browser, such as JAWS 2021.

Use a compatible screen reader with the Firefox browser, such as NVDA 2020.4.

Repeat without a screen reader using only the keyboard (no mouse or touch).

**Expected behavior**

Currently no canonical accessible design pattern exists for split buttons in web pages. A new micro-interaction design may be required to define the expected behavior for screen reader users and sighted keyboard-only users with desktop and mobile browsers.

For example, the following approach would provide more usable interactions for such users:

* Implement each control in this set of related controls as native HTML elements where possible, otherwise one of the canonical [ARIA design patterns](https://www.w3.org/TR/wai-aria-practices-1.1/), so users can understand how to interact without any special instructions.
* Implement the textual button as a simple button element.
* Implement the graphical button and pop-up as a [disclosure pattern](https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure).
* Continue to augment the accessible name of one of the button controls, similar to the addition of "splitbutton" as currently implemented, to convey the relationship between the textual button and the graphical button.

**Browser**

 - OS: Windows 10
 - Browser: Chrome
 - Browser Version: 89
 - Browser: Firefox
 - Browser Version: 87
